config/routes.rb

```diff
@@ -1,4 +1,9 @@
 Rails.application.routes.draw do
   resources :donations, only: :create
-  resources :donors, only: :show
+  resources :donors, only: [:show, :update, :inline_edit] do
```

Comment:
Hm. I haven't seen this naming convention before. I kind of like it, but since it is not adopted in the community yet, let's just name the action `:edit`. WDYT?